### DISCUSSION TITLE
Hidden step name when it matches secret

### DISCUSSION THREAD

**Discussion by RouNNdeL** [community/community#25082] [Actions]
<p>I belive I found a bug related to hiding the secrets from the logs. When my step name contains the secret it gets hidden. Ex. I have a secret containing the string ‘release’. When my step is called ‘Build Release’ it turns into ‘Build ***’. I understand the need to block any form of the secret in the logs, such as ‘app-release.apk’ should be replaced by ‘app-***.apk’, however I do not think it should be the case for hardcoded step names. </p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Sorry to tell you that this is by designed now. All strings which match secret variable in console logs will be masked.</p>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Factions%2Ftoolkit%2Fblob%2F225370fc48622bbffa008d0ad36174dbc7530dee%2Fdocs%2Fcommands.md%23register-a-secret&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7Ccafc93a881ee40550ecd08d79499ac63%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637141260054348596&amp;sdata=Q48tgddEzOOT4iA7GIc6mDrcNZkskFrfHld3TsGqZa8%3D&amp;reserved=0" rel="nofollow noopener">https://github.com/actions/toolkit/blob/225370fc48622bbffa008d0ad36174dbc7530dee/docs/commands.md#register-a-secret</a></p>
